outputs.tf

```
@@ -4,41 +4,41 @@ output "igw_id" {
 }
 
 output "vpc_id" {
-  value       = aws_vpc.default.id
+  value       = aws_vpc.default.0.id
```

```suggestion
  value       = join("", aws_vpc.default.*.id)
```

please use `spalt+join` pattern for consistency (we use it in all our modules).
also `default.0.id` could fail in some cases (at least it did with older TF versions)